doc/source/cluster/kubernetes/configs/static-ray-cluster-networkpolicy.yaml

```diff
@@ -167,137 +47,49 @@ spec:
     - Egress
   egress:
   - to:
-    - podSelector: {}
+    - podSelector:
+        matchLabels:
+          app: redis
     ports:
     - protocol: TCP
       port: 6379
   - to:
-    - podSelector: {}
-    ports:
-    - protocol: TCP
-      port: 6380
-  - to:
-    - podSelector: {}
-    ports:
-    - protocol: TCP
-      port: 8265 
-  - to:
-    - podSelector: {}
-    ports:
-    - protocol: TCP
-      port: 8076
-  - to:
-    - podSelector: {}
-    ports:
-    - protocol: TCP
-      port: 8077
-  - to:
-    - podSelector: {}
-    ports:
-    - protocol: TCP
-      port: 8078
-  - to:
-    - podSelector: {}
-    ports:
-    - protocol: TCP
-      port: 52365
-  - to:
-    - podSelector: {}
-    ports:
-    - protocol: TCP
-      port: 10001
-  - to:
-    - podSelector: {}
-    ports:
-    - protocol: TCP
-      port: 10002
-  - to:
-    - podSelector: {}
-    ports:
-    - protocol: TCP
-      port: 10003
-  - to:
-    - podSelector: {}
-    ports:
-    - protocol: TCP
-      port: 10004
-  - to:
-    - podSelector: {}
-    ports:
-    - protocol: TCP
-      port: 10005
-  - to:
-    - podSelector: {}
-    ports:
-    - protocol: TCP
-      port: 10006
-  - to:
-    - podSelector: {}
-    ports:
-    - protocol: TCP
-      port: 10007
-  - to:
-    - podSelector: {}
-    ports:
-    - protocol: TCP
-      port: 10008
-  - to:
-    - podSelector: {}
-    ports:
-    - protocol: TCP
-      port: 10009
-  - to:
-    - podSelector: {}
-    ports:
-    - protocol: TCP
-      port: 10010
-  - to:
-    - podSelector: {}
-    ports:
-    - protocol: TCP
-      port: 10011
-  - to:
-    - podSelector: {}
-    ports:
-    - protocol: TCP
-      port: 10012
-  - to:
-    - podSelector: {}
-    ports:
-    - protocol: TCP
-      port: 10013
-  - to:
-    - podSelector: {}
-    ports:
-    - protocol: TCP
-      port: 10014
-  - to:
-    - podSelector: {}
-    ports:
-    - protocol: TCP
-      port: 10015
-  - to:
-    - podSelector: {}
-    ports:
-    - protocol: TCP
-      port: 10016
-  - to:
-    - podSelector: {}
-    ports:
-    - protocol: TCP
-      port: 10017
-  - to:
-    - podSelector: {}
-    ports:
-    - protocol: TCP
-      port: 10018
-  - to:
-    - podSelector: {}
-    ports:
-    - protocol: TCP
-      port: 10019
+    - podSelector:
+        matchLabels:
+          app: ray-cluster-worker
+
+---
+# Ray Worker Ingress
```

Comment:
will this open every ports?

Reply:
Yes, it will allow all ports, but just between the head and workers.